### DISCUSSION TITLE
Automatic Selection of New iCloud Albums or Negative Selection Option

### DISCUSSION THREAD

**Discussion by Tonitom** [immich-app/immich#13408] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I’m not sure if this is the right place for the request, but it would be great if the iOS app automatically selected new iCloud albums when they are created. Alternatively, there could be an option for “negative selection” where all albums are selected by default except for certain ones that the user chooses to exclude.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
You can select `Recent` and then double-tap albums to exclude them.

**Reply by Tonitom** [reply]
Unfortunately, I haven't found a way to automatically add new albums in the iOS app. I have over 200 albums, all of which I initially selected manually. The problem is new albums that I will create in Apple Photos in the future.

**Reply by bo0tzz** [reply]
I was responding to the second part of your post ;)
